src/lib/eina/eina_thread.c

```diff
@@ -228,30 +130,46 @@ EAPI Eina_Bool
 eina_thread_cancel(Eina_Thread t)
 {
    if (!t) return EINA_FALSE;
+   #ifndef _WIN32
    return pthread_cancel((pthread_t)t) == 0;
+   #else
+   return _eina_thread_cancel(t);
+   #endif
```

Comment:
I suggest to implement this in `_eina_thread_cancel` (in `eina_thread_win32.x` and `eina_thread_posix.x`), since it is already for these plataform-specific implementations.